cnf-certification-test/identifiers/identifiers.go

```diff
@@ -220,6 +221,22 @@ func InitCatalog() map[claim.Identifier]claim.TestCaseDescription {
 		},
 		TagExtended)
 
+	TestProjectedVolumeServiceAccountIdentifier = AddCatalogEntry(
+		"Projected-Volume-Service-Account",
```

Comment:
I think this should be all lower-case.

Reply:
I would also add -token at the end, same for TestProjectedVolumeServiceAccountIdentifier => TestProjectedVolumeServiceAccountTokenIdentifier